kubernetes/elastic-helm/daemonset.yaml

```diff
@@ -47,3 +47,22 @@ extraEnvs:
 configMap:
   create: false
   existingName: elastic-otelcol-agent-ds
+
+# kubeletstats additional rules for Node metrics
+clusterRole:
+  create: true
+  rules:
+    - apiGroups:
+        - ""
+      resources:
+        - nodes/proxy
```

Comment:
Is this merged with any pre-existing rule defined by the preset?

I think in the final clusterRole we need it to be `resources: [ "nodes/stats", "nodes/proxy"]` 🤔 ?
Checking the output of a `dry-run` of the `helm install` command would help here.

Reply:
> Is this merged with any pre-existing rule defined by the preset?

Yes, these are the resulting rules:

```
rules:
  - apiGroups:
    - ""
    resources:
    - nodes/proxy
    verbs:
    - get
  - apiGroups:
    - ""
    resources:
    - nodes
    verbs:
    - get
    - watch
    - list
  - apiGroups: [""]
    resources: ["pods", "namespaces"]
    verbs: ["get", "watch", "list"]
  - apiGroups: ["apps"]
    resources: ["replicasets"]
    verbs: ["get", "list", "watch"]
  - apiGroups: ["extensions"]
    resources: ["replicasets"]
    verbs: ["get", "list", "watch"]
  - apiGroups: [""]
    resources: ["nodes/stats"]
    verbs: ["get", "watch", "list"]
```

`nodes/stats` is defined with the preset: https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/templates/clusterrole.yaml#L48

If you prefer and to make it more explicit, we could include the `node/stats` rule in this config too. Although there will be a duplicated rule, it should not be an issue.

Reply:
That's fine as is! Thank's for clarifying!